source/_integrations/private_ble_device.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: Private BLE Device
+description: Instructions on how to integrate private BLE devices into Home Assistant.
+ha_category:
+  - Device Tracker
+ha_release: '2023.10'
+ha_iot_class: Local Push
+ha_domain: private_ble_device
+ha_platforms:
+  - device_tracker
+ha_bluetooth: true
+ha_config_flow: true
+ha_integration_type: integration
+ha_codeowners:
+  - '@Jc2k'
+---
+
+Some BLE devices use a privacy feature called Resolvable Private Addresses to stop businesses tracking you when you are out with your smart devices. Instead of having a single static address that can be used to track you, its Bluetooth address changes frequently. If you have [Bluetooth](/integrations/bluetooth) enabled and functioning and you know the Identity Resolving Key for your device, you can add it to Home Assistant as a Private BLE Device. We can then work out which random mac addresses are related to the IRK and track those.
```

Comment:
```suggestion
Some BLE devices use a privacy feature called Resolvable Private Addresses to stop businesses from tracking you when you are out with your smart devices. Instead of having a single static address that can be used to track you, its Bluetooth address changes frequently. If you have [Bluetooth](/integrations/bluetooth) enabled and functioning and know your device's Identity Resolving Key, you can add it to Home Assistant as a Private BLE Device. We can then work out which random MAC addresses are related to the IRK and track those.
```